### DISCUSSION TITLE
Which app launcher do you use?

### DISCUSSION THREAD

**Discussion by JustSimplyKyle** [hyprwm/Hyprland#422] [General]
In sway, I use ulauncher, but after I came to hyprland, I started to look for alternatives.
First I tried yofi, doesn't support my input method.
Later I tried fuzzel, which also doesn't.
Last I tried sirula, it works! But it's a bit slow and has this problem https://github.com/hyprwm/Hyprland/issues/414
So I wonder, what app launchers do you guys use?

**Comment by vars1ty** [comment]
I use wofi since ULauncher is messy on Hyprland.

**Reply by JustSimplyKyle** [reply]
Found a new one that "works" on hyprland(had to resort to xorg mode to make my input method work :sad:)
https://github.com/oknozor/onagre

**Comment by guttermonk** [comment]
I used the Rofi and Wofi listed in the [hyprland wiki](https://wiki.hyprland.org/Useful-Utilities/App-Launchers/). No real issues with either, but I'm curious if there's one that's more performant than the others.

**Comment by aphelei** [comment]
https://codeberg.org/dnkl/fuzzel
